source/features/cross-deleted-pr-branches.tsx

```diff
@@ -16,20 +16,19 @@ function init(): void | false {
 		return false;
 	}
 
-	const deletedBranchName = lastBranchAction.textContent!.trim();
-
 	const headReferenceLink = select('.head-ref a')!;
-	const repoRootUrl = headReferenceLink.href.split('/', 5).join('/');
-	const repoIsDeleted = headReferenceLink.textContent === 'unknown repository';
+	if (!headReferenceLink) {
+		return; // Don't return false, if the repo is deleted the CSS will cross it out
```

Comment:
@fregante can you help me with the wording?

Reply:
```suggestion
		return; // Don't return false; This feature’s CSS already takes care of this
```

Reply:
<img width="685" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/114258778-3d54c380-99f3-11eb-81c4-1260a75e727e.png">

This doesn't seem to be the case. CSS covers the header, but not the body.

Test: https://github.com/sindresorhus/refined-github/pull/271

Reply:
Should be fixed now.